integration/tests/lifecycle_nitrorpc_v02.test.ts

```diff
@@ -282,7 +285,8 @@ describe('nitrorpc_v02 lifecycle', () => {
     });
 
     it('should close channel and withdraw with app funds', async () => {
-        const msg = await createCloseChannelMessage(bob.messageSigner, bobChannelId, bob.walletAddress);
+        // Use wallet signer for channel operations, not session key
+        const msg = await createCloseChannelMessage(bob.walletMessageSigner, bobChannelId, bob.walletAddress);
```

Comment:
I think it would be great to rename several `Identity` fields to better reflect the "sessionKey" - "wallet" substantial difference introduced in this PR.
I suggest renaming `messageSigner` to `messageSKSigner` and `walletMessageSigner` to `messageWalletSigner`, the same for `stateSigner` and `walletStateSigner`, and also rename `sessionAddress` to `sessionKeyAddress`.
I believe such changes are the most relevant to this PR.